_includes/components/choice-menu-items.md

```diff
@@ -0,0 +1,28 @@
+#### Choice menu items
+
+Choice menu items are a special type of menu item used for split toolbar button menu items. For information on split buttons, see: [Split toolbar buttons]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/#splitbutton).
+
+##### Config options
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| text | string | optional | Text to display. |
+| icon | string | optional | Name of the icon to be displayed. Must correspond to an icon in the [icon pack]({{site.baseurl}}/advanced/editor-icon-identifiers/) or an icon in a [custom icon pack]({{site.baseurl}}/advanced/creating-an-icon-pack/). |
```

Comment:
Have just realised that we don't mention registered icons in these things? E.g. adding an icon specifically via editor.ui.registry.addIcon or whatever it is?

Reply:
Fixed everywhere using a variable.
May have been fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/67539b3f222703c4de3af5e10a6c9848d82ea612